app/validators.R

```diff
@@ -18,6 +18,6 @@ validatorOwnCromwell <- function(x) {
   )
   # URL must be a valid URL via rcromwell::cromwell_version
   iv$add_rule("ownCromwellURL",
-    ~ if (!check_url(.)) "Not a valid Cromwell URL")
+    ~ if (!check_url(.)) "Not a valid URL")
```

Comment:
But the `check_url` function isn't validating URLs in general, just that it's a valid cromwell server. So a valid url is a little misleading, but if  you're just going for simpler then I guess thats fine

Reply:
Either way is fine! But this would be an update i dont think for the screenshots though...

Reply:
Perhaps HERE is where we write "Not a valid server URL" adn leave it silent as to whether it's a PROOF based server or a DIY Cromwell server?

Reply:
Sure sounds good